src/EbmlString.cpp

```diff
@@ -64,7 +64,7 @@ filepos_t EbmlString::ReadData(IOCallback & input, ScopeMode ReadFully)
   } else {
     Value.resize(GetSize());
     std::fill_n(Value.begin(), GetSize(), 0);
-    input.readFully(&Value[0], GetSize());
+    input.readFully(const_cast<char*>(Value.data()), GetSize());
```

Comment:
The const cast is no longer needed in C++17 https://en.cppreference.com/w/cpp/string/basic_string/data

Reply:
Makes sense.

Reply:
CMakeLists.txt still specifies 14.